charts/mothership/templates/bundler.yaml

```diff
@@ -0,0 +1,80 @@
+{{ range $idx := until (int $.Values.bundler.count) }}
+{{ $index := add $idx 1 }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: stackup-bundler
+  namespace: {{ $.Release.Namespace }}
+spec: 
+  selector:
+    matchLabels:
+      name: stackup-bundler
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: stackup-bundler
+    spec:
+      initContainers:
+        - name: wait-for-op-geth
+          image: curlimages/curl
+          command: ['sh', '-c']
+          args:
+            - |
+              until curl -sf {{ $.Values.bundler.gethRpc }};
+                do echo waiting for op-geth;
+                sleep 5;
+              done
+      containers:
+        - name: stackup-bundler
+          image: {{ $.Values.bundler.image }}
+          ports:
+            - name: stackup-bundler-rpc
+              containerPort: 4337
+          env:
+            - name: ERC4337_BUNDLER_ETH_CLIENT_URL
+              value: {{ $.Values.bundler.gethRpc }}
+            - name: ERC4337_BUNDLER_SUPPORTED_ENTRY_POINTS
+              value: {{ $.Values.bundler.entryPointContract }}
+            - name: ERC4337_BUNDLER_PRIVATE_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: private-keys
+                  key: ERC4337_BUNDLER_KEY
+            - name: ERC4337_BUNDLER_NATIVE_BUNDLER_COLLECTOR_TRACER
+              value: "bundlerCollectorTracer"
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: bundler-{{ $index }}
+  namespace: {{ $.Release.Namespace }}
+  annotations:
+    {{- if $.Values.bundler.loadBalancerExternal }}
+    service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
+    service.beta.kubernetes.io/aws-load-balancer-type: external
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: {{ $.Values.awsLoadBalancerSslCert }}
+    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
+    {{- end }}
+spec:
+  selector:
+    name: bundler-{{ $index }}
```

Comment:
selector should match the labels of statefulset to route traffic correctly. so in this case selector should be `name: stackup-bundler` followed by sts above.
also i think we need to use `{{ $index }}` for sts above

Reply:
I added missing keyword and modified service name label.
thank you